components/function-controller/internal/controllers/knative/service_controller_test.go

```diff
@@ -210,11 +261,79 @@ func fixRevisionList(parentSvcName, namespace string, num int) []servingv1.Revis
 	return revList
 }
 
-func fixKservice(name, namespace string) servingv1.Service {
+func fixKservice(name, namespace string, addLables bool) servingv1.Service {
+	svcLabels := map[string]string{
+		testCleanupLabelKey: testCleanupLabelValue,
+	}
+
+	if addLables {
+		svcLabels = map[string]string{
+			serverlessv1alpha1.FunctionNameLabel:      "whatever-name",
+			serverlessv1alpha1.FunctionUUIDLabel:      "whatever-uuid",
+			serverlessv1alpha1.FunctionManagedByLabel: "whatever-managed-by",
+			testCleanupLabelKey:                       testCleanupLabelValue,
```

Comment:
Maybe push new labels, not override whole map? I see that in above lines you create default map with only this label.